nautobot/extras/templates/extras/scheduled_jobs_approval_queue_list.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'generic/object_list.html' %}
+
+{% block title %}Jobs Approval Queue{% endblock title %}
+
+{% block javascript %}
+{{ block.super }}
+<script>
+    function handleDetailPostAction(url, action) {
```

Comment:
Comment here explaining what this script is doing and why would be appreciated. :-)